services/web/server/src/simcore_service_webserver/users/schemas.py

```diff
@@ -104,3 +105,26 @@ def to_capitalize(cls, v):
         if isinstance(v, UserRole):
             return v.name.capitalize()
         return v
+
+
+#
+# helpers
+#
+
+
+def convert_user_db_to_schema(
+    row: Mapping[str, Any], prefix: str | None = ""
+) -> dict[str, Any]:
+    parts = row[f"{prefix}name"].split(".") + [""]
```

Comment:
`prefix=None` is not handled here it will just be serialised to `'None'` string. Is this OK?

Reply:
I see this was always the case

Reply:
it is indeed a bug. I will check. thx for pointing out